src/stdune/temp.ml

```diff
@@ -1,16 +1,69 @@
+let try_times n ~f =
+  assert (n > 0);
+  let rec loop n =
+    if n = 1 then
+      f n
+    else
+      match f n with
+      | exception _ -> loop (n - 1)
+      | r -> r
+  in
+  loop n
+
+let prng = lazy (Random.State.make_self_init ())
+
+let temp_file_name ?temp_dir ~prefix ~suffix () =
+  let temp_dir =
+    match temp_dir with
```

Comment:
This doesn't seem necessary; this function is called only once and without a `?temp_dir` argument. It also feel that we should obtain the temporary directory outside of the loop.

Reply:
Removed.